.github/workflows/cherry-pick-to-frozen.yml

```
@@ -0,0 +1,276 @@
+          SOURCE_PR_URL: ${{ github.event.pull_request.html_url }}
+          SOURCE_PR_NUMBER: ${{ github.event.pull_request.number }}
+          SOURCE_PR_TITLE: ${{ github.event.pull_request.title }}
+          BASE_BRANCH: ${{ needs.prepare.outputs.base_branch }}
+          ADD_CP_TO_TRUNK_NAG: ${{ needs.cherry-pick-to-trunk-check.outputs.add_cp_to_trunk_nag }}
+          NEXT_BRANCH: ${{ needs.prepare.outputs.next_branch }}
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.CODE_FREEZE_BOT_TOKEN }}
+          slack-channel: ${{ secrets.WOO_RELEASE_SLACK_NOTIFICATION_CHANNEL }}
+          slack-optional-unfurl_links: false
+          slack-text: |
+            :warning: *Cherry pick from `${{ env.BASE_BRANCH }}` to `trunk` <${{ env.WORKFLOW_RUN_URL }}|failed>*  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)